scripts/qmflows/convolution.py

```diff
@@ -47,25 +47,25 @@ def main(sigma, n, nm, write):
        y_grid = convolute(data[:, 0], data[:, 1], x_grid, sigma)
 
     #Convert in nm if request
-    if nm:
+    if nm == True:
```

Comment:
The previous syntax is the right one: ` if nm` you don't need to to declared `if nm ==True`